### PR TITLE
CLOUD-612 Testing default cr only on release branch

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Review by tplavcic — APPROVED** [review]

File: cloud/jenkins/pxc_operator_gke_version.groovy [hunk 1]
```diff
@@ -304,6 +304,12 @@ pipeline {
                 stage('E2E Basic Tests') {
                     steps {
                         CreateCluster('basic')
+                        stage('E2E Default CR') {
```
Comment (hors) [inline cloud/jenkins/pxc_operator_gke_version.groovy:307]: could you please test it

Comment (cap1984) [reply cloud/jenkins/pxc_operator_gke_version.groovy]: done. I've rearranged the code

File: cloud/jenkins/pxc_operator_gke_version.groovy [hunk 2]
```diff
@@ -302,23 +302,42 @@ pipeline {
...
-                        ShutdownCluster('basic')
-                   }
+                    stages {
+                        stage('cluster creation') {
+                            steps {
+                                CreateCluster('basic')
+                            }
+                        }
+                        stage('default CR') {
+                            when { expression { params.GIT_BRANCH.contains('release-') }}
```
Comment (hors) [inline cloud/jenkins/pxc_operator_gke_version.groovy:312]: maybe it is better to add this logic into `runTest` function

**Review by hors — CHANGES_REQUESTED** [review]

File: cloud/jenkins/pxc_operator_gke_version.groovy (hunk 2 above)
Comment (cap1984) [reply cloud/jenkins/pxc_operator_gke_version.groovy:312]: I've created a wrapper function. Please check

**Review by hors — APPROVED** [review]

**Pull request merged**